bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -286,7 +286,8 @@ async fn migrate_data(
     let tracked_users: Vec<_> = data
         .tracked_users
         .into_iter()
-        .map(|u| Ok(((parse_user_id(&u)?), true)))
+        .flat_map(|s| parse_user_id(&s))
+        .map(|u| Ok((u, true)))
         .collect::<anyhow::Result<_>>()?;
```

Comment:
Let's make it clearer that the collect will now never fail:

```suggestion
        .filter_map(|s| parse_user_id(&s).ok().map(|u| (u, true)))
        .collect();
```